generation/update_owlbot_config.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# If generated samples directory doesn't exist yet, add it to the owlbot config
+
+# Usage:
+# update_owlbot_config.sh <.OwlBot.yaml path from the root>
+# Example:
+# $ update_owlbot_config.sh java-dataform/.OwlBot.yaml
+#
+# To apply the change to all OwlBot configuration files in all modules:
+# $ for F in `find . -maxdepth 2 -name '.OwlBot.yaml'`; do sh generation/update_owlbot_config.sh $F; done
+
+OWLBOT_FILE=$1
```

Comment:
We already have `set_owlbot_config.sh`. Can we just update that script instead of adding another one?

Reply:
That makes sense. I kept it separately initially for testing purposes, but I didn't run into any issues combining.  I've merged the two into `set_owlbot_config.sh`.